app/assets/stylesheets/aspects.scss

```diff
@@ -4,20 +4,26 @@
     @include selectable-list;
 
     .status_indicator {
-      width: 19px;
       height: 14px;
+      width: 19px;
```

Comment:
You reverted your change here, but please revert the order change too, because it is not related with this PR.